CODEOWNERS

```diff
@@ -4,24 +4,71 @@
 
 # Each line is a file pattern followed by one or more owners.
 # Please feel free to add yourself to a module or create a new mapping.
+# Ideally each module should have two or more owners.
 
-# These owners will be the default owners for everything in the repo.
-# Later match takes precedence
+# Code owners are automatically requested for review
+# when someone opens a pull request that modifies code that they own.
+# Later matches take precedence.
 
-std/bitmanip.d @WalterBright
-std/compiler.d @WalterBright
+etc/c/* @CyberShadow
+std/algorithm/* @andralex @wilzbach
+#std/array.d
+#std/ascii.d
+#std/base64.d
+#std/bigint.d
+#std/bitmanip.d
+std/c/windows/* @CyberShadow
+#std/compiler.d
+#std/complex.d
 std/concurrency.d @MartinNowak
+#std/container/
+std/conv.d @JackStouffer
+#std/csv.d
 std/datetime/* @jmdavis
+#std/demangle.d
 std/digest/* @jpf191
+#std/encoding.d
+#std/exception.d
 std/experimental/allocator/* @andralex
 std/experimental/checkedint/* @andralex
 std/experimental/logger/* @burner
+#std/experimental/typecons.d
+#std/file.d
+#std/format.d
+#std/functional.d
+#std/getopt.d
+#std/internal/
+std/json.d @CyberShadow
 std/math* @don-clugston-sociomantic @WalterBright
+#std/math.d
+#std/mathspecial.d
 std/meta.d @klickverbot @ZombineDev
+#std/mmfile.d
+std/net/curl.d @CyberShadow
+#std/net/isemail.d
+#std/numeric.d
+#std/outbuffer.d
+#std/parallelism.d
+std/path.d @CyberShadow
 std/process.d @kyllingstad @schveiguy @CyberShadow
 std/random.d @WebDrake @wilzbach
+std/range/* @andralex @jmdavis @wilzbach
```

Comment:
Ah, okay. That's fine. I already get e-mails for every PR creation and comment anyway. So, if anything, the problem is not having key things buried rather than not being notified. Fortunately, I recently got it fixed so that my e-mail client is actually able to single out the e-mails that include `@jmdavis`, since my filter for that wasn't working before. And without a working filter for that sort of thing, I suspect that a number of us risk never seeing a message targeted at us (for anyone who cares and has also had filtering problems, the key thing for my e-mail client was to watch for the e-mail to be cc-ed to `mention@noreply.github.com` - since for whatever stupid reason, it was failing to filter based on the e-mail's content).